libcloudforensics/gcp.py

```diff
@@ -1340,6 +1344,21 @@ def ExportImage(self, gcs_output_folder, output_name=None):
     cloud_build.BlockOperation(response)
     log.info('Image {0:s} exported to {1:s}.'.format(self.name, full_path))
 
+  def Delete(self):
+    """Delete Compute Disk Image from a project.
+
+    Returns:
+      dict: Holding the response of a get operation on an API object of type
+          globalOperations..
+    """
+
+    gce_image_client = self.project.GceApi().images()
+    request = gce_image_client.delete(
+        project=self.project.project_id, image=self.name)
+    response = request.execute()
+    result = self.project.BlockOperation(response)
+    return result
```

Comment:
I see we don't return anything for similar methods (e.g. `GoogleComputeSnapshot.Delete()`). Do we need the response here, or is it missing in the first place in the other method(s)?

Reply:
mmm, since blockOperation will raise errors, we can get rid of this return for the sake of consistency.